lib/db2.js

```diff
@@ -50,6 +51,8 @@ function DB2(settings) {
     ';PORT=' + this.portnumber +
     ';PROTOCOL=' + this.protocol;
 
+  this.connStr = this.dsn || this.connStrGenerate;
+
   this.schema = this.username;
   if (settings.schema) {
     this.connStr += ';CurrentSchema=' + settings.schema;
```

Comment:
@qpresley @bajtos I am not sure if a db2 user always include schema in their `dsn`, I keep the snippet of adding schema since the credential of my dashdb doesn't contain schema:
```
"dsn": "DATABASE=BLUDB;HOSTNAME=ys1-dashdb-small-bm01-im-dal06-env4.services.dal.bluemix.net;PORT=50000;PROTOCOL=TCPIP;UID=dash107584;PWD=K6eSujXEGXYe;",
```